docker/init_debian.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# Note: When modifying this file, increment the Dockerfile-version minor version number
+# This is needed for mob tool to be able to pull the right image from the farm,
+# if it isn't done, bad things will happen to local builds and CI using mob tool
+# See mob tool comments for extended discussion
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+set -e  # Die on any errors
+
+cd /tmp
+
+# Certain Installers make 'installations' easier by having a nice front-end. While this is great when you have a manual install, this becomes an issue during automated installations.
+export DEBIAN_FRONTEND=noninteractive
+
+# Install build tools and dependencies
+apt-get update -q -q
+apt-get upgrade --yes
+apt-get install --yes \
+  alien \
+  apt-transport-https \
+  autoconf \
+  automake \
+  binutils-dev \
+  build-essential \
+  clang \
+  cmake \
+  curl \
+  git \
+  jq \
+  libclang-dev \
+  libcurl4-openssl-dev \
+  libdw-dev \
+  libiberty-dev \
+  libpq-dev \
+  libprotobuf-c-dev \
+  libprotobuf-dev \
+  libssl-dev \
+  libssl1.1 \
+  libsystemd-dev \
+  libtool \
+  libxml2-dev \
+  llvm-dev \
+  nano \
+  nginx \
+  ninja-build \
+  ocaml-native-compilers \
+  ocamlbuild \
+  patch \
+  pkg-config \
+  postgresql \
+  prometheus \
+  protobuf-c-compiler \
+  protobuf-compiler \
+  psmisc \
+  python \
+  python3-pip \
+  sqlite3 \
+  systemd \
+  unzip \
+  uuid-dev \
+  wget \
+  zlib1g-dev
+# psmisc = killall
+# prometheus = helps with running slam scripts locally
+
+# For use in CI
+# TODO: Do we need this outside CI?
+pip3 install awscli black
+
+# filebeat is used for logs when running slam scripts locally
+# via https://www.elastic.co/guide/en/beats/filebeat/current/setup-repositories.html
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
+echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" >> /etc/apt/sources.list.d/elastic-7.x.list
+apt-get update && apt-get install filebeat
+systemctl enable filebeat
```

Comment:
it looks like you are changing the way that filebeat is installed, and configuring it to start on boot.

is there a reason to add `systemctl enable filebeat`?

it seems like this isn't helpful to most developers or to CI or to the release scripts, so we should not add this and keep it like it was?

Reply:
Good catch. I removed the `enable filebeat` call.